gtsfm/view_graph_estimator/view_graph_estimator_base.py

```diff
@@ -43,6 +43,8 @@ def run(
         corr_idxs_i1i2: Dict[Tuple[int, int], np.ndarray],
         keypoints: List[Keypoints],
         two_view_reports: Dict[Tuple[int, int], TwoViewEstimationReport],
+        cameras_gt: Optional[List[PinholeCameraCal3Bundler]] = None,
```

Comment:
@johnwlambert @ayushbaid we should decide on some design rules here. I think passing gt_cameras to run() beats the purpose of having separate run() and compute_metrics() methods. Its already doing that a little with passing two_view_reports. That was a hack to plot the graphs that John wanted. We should probably not separate these methods if we are computing metrics in run().